src/healthkit.cs

```diff
@@ -3099,6 +3163,16 @@ interface HKUnit : NSCopying, NSSecureCoding {
 		[Static]
 		[Export ("degreeAngleUnit")]
 		HKUnit DegreeAngle { get; }
+
+		[Watch (10, 0), MacCatalyst (17, 0), Mac (14, 0), iOS (17, 0), NoTV]
+		[Static]
+		[Export ("luxUnitWithMetricPrefix:")]
+		HKUnit CreateLuxUnit (HKMetricPrefix prefix);
```

Comment:
Sorry, I got confused, _this_ matches (most of) the other methods in this class:

```suggestion
		HKUnit CreateLux (HKMetricPrefix prefix);
```

Reply:
@rolfbjarne ah yep I missed that too, fixed